static/beta/prod/navigation/rhel-navigation.json

```diff
@@ -39,7 +39,33 @@
             "groups",
             "Group",
             "Groups"
-          ]          
+          ],
+          "permissions": [
+                {
+                  "method": "featureFlag",
+                  "args": ["platform.rbac.groups-to-workspaces-rename", false]
+                }
+              ]
+        },
+        {
+          "id": "workspaces",
+          "appId": "inventory",
+          "title": "Workspaces",
+          "href": "/insights/inventory/workspaces",
+          "product": "Red Hat Insights",
+          "description": "Group your Red Hat Enterprise Linux systems for more granular User Access.",
+          "alt_title": [
+            "workspace",
+            "workspaces",
+            "Workspace",
+            "Workspaces"
+          ],
```

Comment:
nit-pick: this would be nice to preserve "group"-like alternative titles so that users can still be suggested with Workspaces even though they searched for outdated "groups" keyword